dashboard/app/assets/stylesheets/projects.scss

```diff
@@ -17,81 +17,3 @@
     }
   }
 }
```

Comment:
Can we delete this whole file?

Reply:
good catch! I left it because I saw that `projects-page` was still in use in the codebase, but it looks like we render into it using react, and do not accidentally rely on any of these styles. Removed!